nvc.1

```diff
@@ -667,14 +667,30 @@ a coverage bin for each state of an FSM. NVC considers internal signals of
 all user-defined enum types as FSMs. NVC does not consider port signals or
 variables as an FSM. When a signal recognized as FSM changes its value,
 coverage bin for new state value is incremented.
+
 .It
 .Cm functional
-- NVC creates a coverage bin for each PSL
+- NVC creates a coverage bin for each:
+.Bl -bullet
+.It
+PSL
 .Ql cover
-directive. When a cover sequence completes, coverage bin is incremented.
+directive. When a PSL sequence in the cover directive completes, coverage bin is incremented.
+.It
+OSVVM functional coverage bin. A call to
+.Ql AddBins
+or
+.Ql AddCross
+OSVVM functions creates such NVC functional coverage bins.
+A threshold for OSVVM coverage bin to be reported as covered is given by
+.Ql AtLeast
+argument of
+.Ql AddBins
+OSVVM call.
```

Comment:
Do not mention OSVVM here. As I said before this should be a generic API and not something specific to OSVVM.

Reply:
I have left just a note that user gets a hint the integration is supported.